types/src/execution_result.rs

```diff
@@ -378,15 +435,31 @@ pub enum OpKind {
     NoOp,
 }
 
-impl ToBytes for OpKind {
-    fn to_bytes(&self) -> Result<Vec<u8>, bytesrepr::Error> {
+// TODO(sam)
+// See if reviewer thinks these types should be combined --
+// might more content be added to OpKind later on? I'd hope
+// not.
```

Comment:
We will most likely be getting rid of `Op`-like types and operations so it's fine to not put more work into combining them.